instructions/.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea/fenrir-intro.iml
+.idea/modules.xml
+.idea/vcs.xml
+idea
+
```

Comment:
The main .gitignore is usually in the top level directory and affects everything in the hierarchy.